src/chocolatey/infrastructure.app/configuration/ChocolateyConfiguration.cs

```diff
@@ -497,6 +506,17 @@ public sealed class ApiKeyCommandConfiguration
     }
 
     [Serializable]
+    public sealed class PackCommandConfiguration
+    {
+        public PackCommandConfiguration()
+        {
+            Properties = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);

```

Comment:
I get that. Not used to see it in a PR (we generally use comments for either asking about code or asking for a change). I have our internal build defs set up to even require resolving all comments one way or another, so it just seemed a bit odd.